Libraries/LibWeb/HTML/HTMLSelectElement.cpp

```diff
@@ -377,7 +377,7 @@ String HTMLSelectElement::value() const
 WebIDL::ExceptionOr<void> HTMLSelectElement::set_value(String const& value)
 {
     update_cached_list_of_options();
-    for (auto const& option_element : m_cached_list_of_options)
+    for (auto const& option_element : list_of_options())
```

Comment:
Whoops, my bad 😅  The code here is super confusing with how the options are managed (since options call back up to the parent select all over the place and do things in a really slow manner if following spec to the letter), but do you happen to have a javascript reproducer for this? Was it flaking somewhere, or is it just the pattern you noticed?

The more I look into this function the more I'm thinking of changing it